help/en/html/tools/fastclock/index.shtml

```diff
@@ -47,10 +47,11 @@
       provides the basic time information, while one or more
       displays show you what's going on.</p>
 
-      <h2>Fast Clock Timebase</h2>To create a Timebase, use the
+      <h2>Fast Clock Timebase</h2>
+      <p>To create a Timebase, use the
       "<a href=
-      "../../../package/jmri/jmrit/simpleclock/SimpleClockFrame.shtml">Setup
-      Fast Clock</a>" item in the "Clocks" submenu under the JMRI
+      "../../../package/jmri/jmrit/simpleclock/SimpleClockFrame.shtml">Set
+      up Fast Clock...</a>" item in the "Clocks" submenu under the JMRI
```

Comment:
It looks like you used a space here ( or a new line ).

Reply:
As noun, changed ``set-up`` to ``setup`` (no hyphen) in all JMRI for consistency. As a verb always 2 separate words ``(to) set up``.